recipe.yml

```diff
@@ -21,6 +21,7 @@ install:
   - nomarkup
   - paragraphs
   - paragraphs_edit
+  - swiper_formatter
```

Comment:
Do we need to import config from the module?

https://git.drupalcode.org/project/swiper_formatter/-/tree/2.0.x/config?ref_type=heads